plugins/inputs/mqtt_consumer/mqtt_consumer.go

```diff
@@ -74,6 +76,7 @@ type MQTTConsumer struct {
 
 	MetricBuffer      int `toml:"metric_buffer" deprecated:"0.10.3;2.0.0;option is ignored"`
 	PersistentSession bool
+	collectBytes      bool   `toml:"collect_bytes_received"`
```

Comment:
As said before, this isn't needed as the user can already choose to enable the `internal` plugin or not.

Reply:
Missed that one, thanks for pointing that out, fixed in most recent commit